translations/monero-core_de.ts

```diff
@@ -2740,9 +2745,9 @@ Spending address index: </source>
     </message>
     <message>
         <location filename="../main.qml" line="823"/>
-        <source>Couldn&apos;t generate a proof because of the following reason: 
```

Comment:
When you edit the english translation, also update it in the other translation files.

Reply:
@Leza89 why the source text was changed here?

Reply:
I don't know. I guess Atom removed the extra space again. I also don't know how to change this behaviour. I can't even remember that I touched this translation.

It also does not show up as a change in my local repository (GitHub Desktop).